src/GameFace/Struct/UserProfile.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GameFace.Struct
+{
+    public static class Helper
+    {
+        public static List<UserRecords> AddElement(List<UserRecords> list, UserRecords element)
+        {
+            var found = false;
+            if (list.Count == 0)
+            {
+                list.Add(element);
+            }
+            else
+            {
+                for (var i= 0; i < list.Count; i++)
+                {
+                    if (!list[i].Task.Equals(element.Task)) continue;
+                    var newrecord = new UserRecords(list[i].Task, list[i].NoPerforms + 1, list[i].Value + element.Value);
+                    list.Remove(list[i]);
+                    list.Add(newrecord);
+                    found = true;
+                }
+                if (!found)
+                {
+                    list.Add(element);
+                }
+            }
+            
+            return list;
+        }
+
+
+        public static List<SprintRecords> DevideBySprints(List<UserRecordsDate> list)
+        {
+            var date = list[0].Date;
+            int quarterNumber = (date.Month - 1) / 3 + 1;
+            var firstDayOfQuarter = new DateTime(date.Year, (quarterNumber - 1) * 3 + 1, 1);
+            var lastDayOfQuarter = firstDayOfQuarter.AddMonths(3).AddDays(-1);
```

Comment:
It is not necessary, but if we wanted to be fancy we could use some extension methods here. It is just syntax sugar, but it does have the benefit that the intention of the code is a bit clearer, and it can be used by any DateTime in the project. An example of what it might look like if we choose to use extension methods:

```csharp
int quarterNumber = date.QuarterNumber();
int firstDayOfQuarter = date.FirstDayOfQuarter();
int lastDayOfQuarter = date.LastDayOfQuarter();
```